CHANGELOG.md

```diff
@@ -0,0 +1 @@
+- Adds support for configuration with multiple storage targets (#4281).
```

Comment:
There are some [docs](https://firebase.google.com/docs/cli/targets#configure_your_firebasejson_file_to_use_deploy_targets), somewhat hidden away...

Reply:
That's true, the emulator-specific docs don't seem to mention this: https://firebase.google.com/docs/emulator-suite/install_and_configure#security_rules_configuration

But it's okay that we don't reiterate everything since it's already in the CLI documentation